proto/src/determined/api/v1/api.proto

```diff
@@ -941,6 +941,17 @@ service Determined {
       tags: "Experiments"
     };
   }
+
+  // Cleanup task logs according to the retention policy.
+  rpc CleanupLogs(CleanupLogsRequest) returns (CleanupLogsResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/cleanup_logs"
```

Comment:
do you want to s/gc_logs/cleanup_logs/g? i think it'd be easy if it was named consistently throughout this change

Reply:
I updated the command to be `cleanup-logs` but it seems to cause an issue with our help formatter (anything over 8 characters does):
```
positional arguments:
  subcommand
    help        show help for this command
    cleanup-logs
                cleanup expired task logs
    config      manage master config
    info        fetch master info
    logs        fetch master logs
```